environment.yml

```diff
@@ -0,0 +1,7 @@
+channels:
+  - conda-forge
+dependencies:
+  - numpy
+  - jupyter-server-proxy
+  - "jupyterlab >0.34.9"
```

Comment:
Do we need to explicitly mention jupyterlab? Should be installed "as default" by repo2docker and should be v1.x so we are good there as well.

Reply:
I just ported the `requiurements.txt` over here, you probably know better than me why this line is in here.

Reply:
Then we should delete it because it is "legacy".